tests/dummy/app/routes/nodes/detail/draft-registrations/detail.js

```diff
@@ -1,72 +1,14 @@
 import Ember from 'ember';
-import permissions from 'ember-osf/const/permissions';
 
 export default Ember.Route.extend({
     model(params) {
         let node = this.modelFor('nodes.detail');
-        let draft = this.store.peekRecord('draft-registration', params.draft_registration_id);
-        return Ember.RSVP.hash({
-            node: node,
-            draft: draft
-        });
+        this.store.adapterFor('draft-registration').set('namespace', 'v2/nodes/' + node.id);
```

Comment:
I can see why there are better places to put this but I don't know where it should go. My reasoning for putting it here is that I needed access to the node. Maybe we need to think bigger picture about this - findRecord is not really setup to locate nested resources. It's fine most of the time, like logs, for example, you can access nodes/<node_id>/logs/ but if you want log detail, you go to logs/log_id.

This doesn't work for drafts because there is no drafts/draft_id endpoint.